NExtends/Primitives/DateTime.extensions.cs

```diff
@@ -152,7 +154,7 @@ public static String ToShortDateTimeString(this DateTime d)
 		/// <returns></returns>
 		public static String ToShortDate(this DateTime d)
 		{
-			return d.ToString("MMM", CultureInfo.CreateSpecificCulture("en-US")) + " " + d.Year;
+			return d.ToString("MMM", new CultureInfo("en-US")) + " " + d.Year;
 		}
 
 		public static String[] dayText = { "Lundi", "Mardi", "Mercredi", "Jeudi", "Vendredi", "Samedi", "Dimanche" };
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'dayText' or make it 'const' or 'readonly'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS2223)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use an immutable collection or reduce the accessibility of this field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS2386)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this field 'private' and encapsulate it in a 'public' property. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1104)